src/goTest.ts

```diff
@@ -0,0 +1,78 @@
+'use strict';
+
+import cp = require('child_process');
+import path = require('path');
+import vscode = require('vscode');
+import util = require('util');
+import { getGoRuntimePath } from './goPath'
+
+/**
+* Executes the unit test at the primary cursor using `go test`. Output
+* is sent to the 'Go' channel.
+* 
+* TODO: go test returns filenames with no path information for failures,
+* so output doesn't produce navigable line references.
+*/
+export function testAtCursor() {
+	var editor = vscode.window.activeTextEditor;
```

Comment:
I think this can be null/undefined (if there is no file open). Should test for that and render warning.